core/src/main/java/com/novoda/noplayer/Player.java

```diff
@@ -225,4 +224,9 @@ public void onLoadTimeout() {
 
         void onLoadTimeout();
     }
+
+    interface HeartbeatCallback {
```

Comment:
Moved from `Heart.Heartbeat.Callback` to match the other callbacks on the `Player`.